content/zh-cn/includes/task-tutorial-prereqs.md

```diff
@@ -11,4 +11,4 @@ or you can use one of these Kubernetes playgrounds:
 -->
 
 * [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)
-* [玩转 Kubernetes](http://labs.play-with-k8s.com/)
+* [玩转 Kubernetes](https://labs.play-with-k8s.com/)
```

Comment:
Do we need to copy and hide the corresponding en text by following our [localization-zh style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#commented-en-text)?